### PR TITLE
Docusaurus canary upgrade + blog post multi-authors

### PR DESCRIPTION
**Pull request by slorber**
Upgrade to Docusaurus beta.5, fix relevant CSS issues, make some CSS overrides more resilient for upgrades.

Using a canary version (it's safe for prod, just beta.5 + a single patch commit)

Use some new supported features:
- authors.yml
- blog post multi-authors
- co-locate blog post images in the blog post folder

cc @Simek @yungsters

### CODE REVIEW

File: website/src/theme/DocItem/index.js [hunk 1]
```diff
@@ -1,115 +0,0 @@
...
-    formattedLastUpdatedAt,
-    lastUpdatedBy,
-    unversionedId,
-  } = metadata;
-  const {pluginId} = useActivePlugin({
-    failfast: true,
-  });
-  const versions = useVersions(pluginId);
-  const activeVersion = useActiveVersion(pluginId);
-  const showVersionBadge = versions.length > 1 && !activeVersion.isLast;
```
Comment (Simek) [inline website/src/theme/DocItem/index.js:48]: `showVersionBadge` line was also altered to hide badge in the current docs in comparison to the unswizzled version.

**Review by Simek — COMMENTED** [review]
Screenshot 2021-08-31 at 16 43 25

Notice banner close "X" is black after the update.

Screenshot 2021-08-31 at 16 46 54

Also padding on mobile did not look right.

File: website/src/theme/DocItem/index.js (hunk 1 above)
Comment (slorber) [reply website/src/theme/DocItem/index.js:48]: fixed with a new option

File: website/src/css/customTheme.scss [hunk 2]
```diff
@@ -399,9 +399,14 @@ html[data-theme="dark"] .community-page {
 
 /* Version warning */
 
-div[class*="docs-wrapper"] .row .col {
-  > .alert--warning {
-    background-color: rgba(100, 215, 255, 0.3);
+.theme-doc-version-banner.alert--warning {
+  background-color: rgba(100, 215, 255, 0.3);
+  border-style: none;
+  color: var(--ifm-font-color-base);
```
[the remark this answers is not in the document]
Comment (slorber) [reply website/src/css/customTheme.scss:405]: Why not, but I like the shadow more 🤪

File: website/src/css/customTheme.scss [hunk 3]
```diff
@@ -568,16 +575,31 @@ html[data-theme="dark"] .avatar__name a {
...
+    --ifm-navbar-sidebar-width: 480px;
+  }
 
   .navbar-sidebar__brand {
     background-color: var(--deepdark);
   }
 
   .navbar-sidebar__items {
-    padding: 12px;
+    /* padding: 12px; */
```
[the remark this answers is not in the document]
Comment (slorber) [reply website/src/css/customTheme.scss]: Restored in another way

File: website/src/css/customTheme.scss [hunk 4]
```diff
@@ -568,16 +575,31 @@ html[data-theme="dark"] .avatar__name a {
...
+
 .navbar-sidebar {
-  background: var(--divider);
-  max-width: 400px;
+  @media (max-width: 480px) {
+    --ifm-navbar-sidebar-width: 100%;
+  }
+  @media (min-width: 481px) {
+    --ifm-navbar-sidebar-width: 480px;
+  }
```
[the remark this answers is not in the document]
Comment (slorber) [reply website/src/css/customTheme.scss]: limiting the navbar-sidebar width to 340px seems enough to me, let me know what you think but that's largely sufficient to render the sidebar and gives more opportunity to render the drawer overlay

**Comment by slorber** [comment]
All issues reported above should be fixed now

**Comment by Simek** [comment]
To fix the ESLint you will need to add the authors file to the `ignorePatterns` or install and configure `eslint-plugin-yaml`.

**Comment by Simek** [comment]
LGTM, only thing which left if to bump to next `beta` release. 🙂 

Also I would like to wait with the merge for the @yungsters approval on the blog authorship.

**Comment by yungsters** [comment]
Blog authorship looks good to me! I also need to add two more people to that blog post, but I'll submit a follow-up PR.

**Review by yungsters — APPROVED** [review]

**Comment by slorber** [comment]
PR updated with beta.6.

Please let me know asap if there's any other problem, as I'm on holiday tomorrow 🤪

**Review by Simek — APPROVED** [review]
LGTM! @slorber Thank you so much for the work you put into the update and reducing the number of swizzled components! 🚀

**Comment by slorber** [comment]
Thanks, great to see this online!

FYI I created a discussion to report customization use-cases so that we try to provide a robust way to achieve those customizations. 

Let me know if you have other complex customization use-cases that you'd like to replace with simpler code here, or if you'd like to add new customizations to RN website:
https://github.com/facebook/docusaurus/discussions/5468

**Pull request merged**